CODEOWNERS

```diff
@@ -52,14 +52,17 @@
 /.tekton/tasks/ec-checks.yaml       @konflux-ci/ec
 
 # renovate groupName=integration
-/task/clair-scan                @konflux-ci/integration-service-maintainers
-/task/clamav-scan               @konflux-ci/integration-service-maintainers
-/task/deprecated-image-check    @konflux-ci/integration-service-maintainers
-/task/fbc-related-image-check   @konflux-ci/integration-service-maintainers
-/task/fbc-validation            @konflux-ci/integration-service-maintainers
-/task/inspect-image             @konflux-ci/integration-service-maintainers
-/task/sbom-json-check           @konflux-ci/integration-service-maintainers
-/task/validate-fbc              @konflux-ci/integration-service-maintainers
+/task/clair-scan                                @konflux-ci/integration-service-maintainers
+/task/clamav-scan                               @konflux-ci/integration-service-maintainers
+/task/deprecated-image-check                    @konflux-ci/integration-service-maintainers
+/task/fbc-related-image-check                   @konflux-ci/integration-service-maintainers
+/task/fbc-validation                            @konflux-ci/integration-service-maintainers
+/task/inspect-image                             @konflux-ci/integration-service-maintainers
+/task/sbom-json-check                           @konflux-ci/integration-service-maintainers
+/task/validate-fbc                              @konflux-ci/integration-service-maintainers
+/task/fips-operator-bundle-check                @konflux-ci/integration-service-maintainers
```

Comment:
Should your gh account be added here for fips related tasks?

Reply:
Or is there a desire to have another GH team to add? (this can be done in a separate PR though)

Reply:
yeah, adding my name will fail the CODEOWNERS check since I do not have merge rights. Let's take care of it in a separate PR